conda/tools/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - setuptools
   host:
     # TODO: Try to remove when this version settles down among other conda-forge packages
-    - openssl <3.2.0a # [osx]
+    - openssl <3.1.5 # [osx]
```

Comment:
@milljm Making this change fixed up the dependency issues seen on CIVET with `moose-dev`.